optuna/study.py

```diff
@@ -304,6 +304,12 @@ def objective(trial):
                 Flag to show progress bars or not. To disable progress bar, set this ``False``.
                 Currently, progress bar is experimental feature and disabled
                 when ``n_jobs`` :math:`\\ne 1`.
+        
```

Comment:
You can pass the CI by removing white spaces.
```suggestion

```

Reply:
I missed it. Thank you! Fixed.